evalml/model_understanding/graphs.py

```diff
@@ -561,11 +563,15 @@ def partial_dependence(pipeline, X, features, grid_resolution=100):
     if pipeline.model_family == ModelFamily.BASELINE:
         raise ValueError("Partial dependence plots are not supported for Baseline pipelines")
 
+    if ((isinstance(features, int) and X.iloc[:, features].value_counts(normalize=True).values[0] + 0.01 > percentiles[1]) or
```

Comment:
we add 0.01 (1%) since SKLearn doesn't allow the percentage of each value to be too close to the percentile (for instance, if a feature contains 95% of a value, SKLearn will still throw the error even if we pass the upper percentile as 0.955). Otherwise, 1% was chosen arbitrarily (compared to 0.0075 or another value)